## Задача №1

**Подкинули монету N раз. Кол-во случаев, когда выпал орёл, на 10% больше, чем кол-во случаев, когда выпала решка. При каком N мы можем сказать, что монета «нечестная» (орёл и решка выпадают с разной вероятностью)?**

А на сколько мы хотим быть уверены? Предполагаю, что эта задача из статистики) Воспользуюсь **гипотезой о вероятности события**. Её критерий:
* U = ((w - p0) * (N**0.5)) / (p0 * q0), где **q0** - 1 - p0


Критическая оценка для него определяет по таблице Лапласса:
* fi(Ucr) = (1 - 2a) / 2   -  для правостороннего случая. a - в моём случае 0.01

Я хочу определить точку при которой наш критерий пересечёт критическое значение. Подставив критическое значение вместо критерия в формулу, я смогу узнать необходимое количиство испытаний. 

Минуя преобразования, сразу запишу формулу:

In [7]:
fi = 2.06 #определил по таблице Лапласса по значению 0.01
p0 = .5 #Априорная вероятность выпадение орла
w = .52 #Указал примерное значение в случае 10%

n = (fi / (2*(w - p0))) ** 2
n    

2652.2499999999955

**Ответ: Примерно 2652 и один неудачный бросок.**

## Задача №2

**Есть две монеты: «правильная» (с одной стороны орёл, с другой решка) и «неправильная» (орёл на обеих сторонах). Берём одну монету наугад и подкидываем 5 раз. Все 5 раз выпадает орёл. Какова вероятность того, что выбрали «правильную» монету?**


Не вижу подвоха у этой задачи. События должны наступить подряд зависимо. Вероятность выпадения орла на правильной монетки - 1/2. Вероятность взять правильную монетку - 1/2. Следовательно: 

In [8]:
(0.5)**6

0.015625

**Ответ: 0.015625**